language/vm/vm_runtime/src/foreign_contracts.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! This file contains models of the vm crate's dependencies for use with MIRAI.
+
+pub mod types {
+    pub mod transaction {
+        pub const MAX_TRANSACTION_SIZE_IN_BYTES: usize = 4096;
```

Comment:
Why not just define this as a u64?

Reply:
This is a foreign contract for that constant.